.github/pull_request_template.md

```diff
@@ -1,27 +1,21 @@
 #### Pull Request Checklist
 
+- [ ] All changes are Go version 1.15 compliant
+- [ ] Your PR passes all CI checks
 - [ ] If this is your first time contributing, we recommend you read the [Code
   Contribution Guidelines](https://github.com/lightningnetwork/lnd/blob/master/docs/code_contribution_guidelines.md)
-- [ ] All changes are Go version 1.12 compliant
-- [ ] The code being submitted is commented according to [Code Documentation and Commenting](https://github.com/lightningnetwork/lnd/blob/master/docs/code_contribution_guidelines.md#CodeDocumentation)
+   - [ ] The code being submitted is commented according to [Code Documentation and
+  Commenting](https://github.com/lightningnetwork/lnd/blob/master/docs/code_contribution_guidelines.md#CodeDocumentation)
+   - [ ] Commits have a logical structure according to [Ideal Git Commit
+  Structure](https://github.com/lightningnetwork/lnd/blob/master/docs/code_contribution_guidelines.md#IdealGitCommitStructure)
 - [ ] For new code: Code is accompanied by tests which exercise both
   the positive and negative (error paths) conditions (if applicable)
-- [ ] For bug fixes: Code is accompanied by new tests which trigger
-  the bug being fixed to prevent regressions
-- [ ] Any new logging statements use an appropriate subsystem and
-  logging level
-- [ ] Code has been formatted with `go fmt`
-- [ ] Protobuf files (`lnrpc/**/*.proto`) have been formatted with
-  `make rpc-format` and compiled with `make rpc`
-- [ ] New configuration flags have been added to `sample-lnd.conf`
+- [ ] For bug fixes: Code is accompanied by new tests which trigger the bug
```

Comment:
nit: I'd rephrase this to "If possible code is accompanied...", since in some cases when code changes a lot it's impossible to create a regression test (for example crash fixes, data races etc.)

Reply:
Makes sense, updated.